.github/ISSUE_TEMPLATE/A.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Having trouble getting started?
+about: Please contact us at support@bugsnag.com for assistance with integrating Bugsnag
+  into your application.
+title: ''
+labels: ''
+assignees: ''
+
+---
+Please checkout our [documentation](https://docs.bugsnag.com/build-integrations/js/#uploading-source-maps) for guides, references and tutorials.
+
+If you have questions about your integration please contact us at [support@bugsnag.com](mailto:support@bugsnag.com).
+
+Alternatively, view additional options at [support.md](../SUPPORT.md).
```

Comment:
The `support.md` file I can see in this PR doesn't isn't in UPPERCASE. Not sure if that matters?

Reply:
Looks to display ok for https://github.com/bugsnag/homebrew-tap/blob/master/.github/support.md. The link itself is not actually clickable as its only shown in the template when adding an issue. I think we have some more work to do on the template builder tool we use to generate this.